articles/zcash-protocol/chapter_1.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: Introduction
+Status: Done
+Level: "1"
+---
+# Chapter 1: Introduction
+---
+Zcash, an evolution of the Zerocash protocol, bridges the gap between transparent Bitcoin transactions and shielded payments enabled by zk-SNARKs (zero-knowledge succinct non-interactive arguments of knowledge). This document delves into the technical intricacies of Zcash, guiding readers through its concepts, abstract and concrete protocols, network upgrades, and key differentiators from both Bitcoin and Zerocash.
```

Comment:
Replace zk-SNARKs (zero-knowledge succinct non-interactive arguments of knowledge) by just `zk-SNARKs` and link it to the right document